csharp/private/wrappers/.clang-format

```diff
@@ -0,0 +1,14 @@
+# Use the Google style in this project.
+BasedOnStyle: Google
+
+DerivePointerAlignment: false

```

Comment:
What's this?

Reply:
>DerivePointerAlignment (bool)
> If true, analyze the formatted file for the most common alignment of & and *. Pointer and reference alignment styles are going to be updated according to the preferences found in the file. PointerAlignment is then used only as fallback.

Its for cases where people are writing `int& foo` vs `int &foo`. Since we are following google guidelines, they only ask for consistency in a project. Setting this to false should enforce `int& foo` in the project.